docs/podman-build.1.md

```diff
@@ -614,6 +614,14 @@ $ podman build -f dev/Dockerfile https://10.10.10.1/podman/context.tar.gz
 
 registries.conf is the configuration file which specifies which container registries should be consulted when completing image names which do not include a registry or domain portion.
 
+## Troubleshooting
+
+If you are using a useradd command within a Dockerfile with large UID/GID, it will create a huge sparse file `/var/log/lastlog`.  This can cause the build to hang forever.  GO language does not support sparse files correctly, which can lead to some huge files being created.
+
+### Solution
+
+If you are using `useradd` within your build script, you should pass the useradd command the `--no-log-init` field.  This tells useradd to not create the lastlog file.
```

Comment:
I'm struggling with my own wording here.  Perhaps: "you should pass the `--no-log-init` parameter to the useradd command."